pieforproviders/pieforproviders
# Set up i18n in the backend and translate signup and login error messages

# 💅🏼 What issue does this fix?


#185

- Sets up i18n in the backend
- Adds Spanish translations for uniqueness validation error message and Devise messages
- Display signup and login errors in the user's preferred locale

## 🍂 Before You Submit


## 🛷 Deployment Considerations


## 🧵 Steps to set up locally



## 🧳 Steps to test

1. Click **Español** on the top right
1. Sign up for a new account using an existing email address and an existing phone number. You should see the error messages on the email and phone number inputs in Spanish
1. Change the language to English and click on the **SIGN UP** button. You should see the error messages in English
1. Finish the signup process with a unique email address and a unique phone number
1. Log in with the unconfirmed account's credentials
1. You should see "You have to confirm your email address before continuing." in English. Switch the language to Spanish and click **INICIA SESIÓN**. You should see "Tienes que confirmar tu cuenta antes de continuar."

## 🕯 Caveats, concerns
